ansible/server-state-playbooks/nightshade-web/playbook.yml

```diff
@@ -0,0 +1,184 @@
+# Install OMERO.web with a public user on localhost
+
+- hosts: all
+
+  handlers:
+
+  # Handler for nginx
```

Comment:
👍  - I didn't realise I could use things defined 'underneath' the playbook. From programming, I wouldn't have expected global scope for defined functions like that.